ui/app/src/Utils/Types.elm

```diff
@@ -1,20 +1,19 @@
 module Utils.Types exposing (..)
 
-import Http
 import Time
 
 
-type ApiResponse e a
```

Comment:
If I see it correctly we get rid of `ApiResponse`? 👍 for removing an extra layer of complexity.